### PR TITLE
Added documentation for the new 'update license' REST API (forward port from maintenance 3.x)

### PR DESCRIPTION
**Pull request by vojtechtoman**
Added the POST /hazelcast/rest/license endpoint to the
CLUSTER_WRITE group ("Using the REST Endpoint Groups").

Added an example in "License Information / REST".

Documented the behavior and limitations of updating the
license at runtime.

Note: OS/EE code already reviewed and merged.

### CODE REVIEW

File: src/docs/asciidoc/getting_started.adoc [hunk 1]
```diff
@@ -230,9 +230,62 @@ Its output is similar to the following:
...
+"message":"License updated at run time - please make sure to update the license
+in the persistent configuration to avoid losing the changes on restart."}
+```
+
+As the message in the above example indicates, the license is updated only at runtime.
+The persistent configuration of each member needs to be updated manually to ensure that
+the license change is not lost on restart. The same message is logged as a warning in
+each member's log.
+
+It is only possible to update to a license that expires at the same time or after
```
Comment (Serdaro) [inline src/docs/asciidoc/getting_started.adoc]: ... possible to update a license that ...

File: src/docs/asciidoc/getting_started.adoc [hunk 2]
```diff
@@ -230,9 +230,62 @@ Its output is similar to the following:
...
+the current license. The new license must allow the exact same list of features and
+the same number of members.
+
+If, for any reason, updating the license fails on some member (member does not respond,
```
Comment (Serdaro) [inline src/docs/asciidoc/getting_started.adoc]: ...  on some members (....

Comment (vojtechtoman) [reply src/docs/asciidoc/getting_started.adoc]: fixed

File: src/docs/asciidoc/getting_started.adoc (hunk 1 above)
Comment (vojtechtoman) [reply src/docs/asciidoc/getting_started.adoc]: The idea is you are updating to a new license, and that license must expire after the current license.

**Comment by Serdaro** [comment]
Thank you! :)

**Pull request merged**